CHANGELOG.md

```diff
@@ -26,6 +26,8 @@ The format is described in [Contributing notes](CONTRIBUTING.md#changelog-entry-
 * Change style to `variable` for `Layout/EndAlignment` cop ([@r.dubrovsky][])
 * Change style to `with_fixed_indentation` with indentation width 2 for `Layout/AlignParameter` cop ([@r.dubrovsky][])
 * Always ingone hash aligning for key word arguments (cop: `Layout/AlignHash`) ([@r.dubrovsky][])
+* Use prefered variable name in rescued exceptions (cop: `Naming/RescuedExceptionsVariableName`) ([@ula][])
```

Comment:
it should be added to unreleased version (not to 0.1.0). I'll add these changes to `0.2.0` version

Reply:
see https://github.com/datarockets/datarockets-style/pull/45/files#diff-4ac32a78649ca5bdd8e0ba38b7006a1eR9